test/relations/empty.test.ts

```diff
@@ -0,0 +1,18 @@
+import { random } from 'faker'
+import { factory, primaryKey, oneOf } from '@mswjs/data'
+
+it('should not throw error if an entity with relation is created without it', () => {
```

Comment:
I think this case should be added to all relevant `test/relations/*.test.ts` files, because the way oneOf and manyOf relations are handled may differ. Think of it as a spec point in each relationship kind support, not a standalone behavior.